arviz/data/inference_data.py

```diff
@@ -245,6 +247,25 @@ def to_netcdf(self, filename, compress=True, groups=None):
             empty_netcdf_file.close()
         return filename
 
+    @staticmethod
+    def from_json(filename):
+        """Initialize object from a json file.
+
+        Parameters
+        ----------
+        filename : str
+            location of json file
+
+        Returns
+        -------
+        InferenceData object
+        """
+
+        with open(filename, "r") as file:
+            idata_dict = json.load(file.read())
```

Comment:
We should check would it be faster to read json in `rb` vs `r`

(Probably doesn't matter, ujson is muchmuch faster)